setup.cfg

```diff
@@ -1,2 +1,52 @@
+[metadata]
+name = metakb
+author = VICC
+author_email = help@cancervariants.org
+description = A search interface for cancer variant interpretations assembled by aggregating and harmonizing across multiple cancer variant interpretation knowledgebases.
+long_description = file:README.md
+long_description_content_type = text/markdown
+home-page = https://github.com/cancervariants/metakb
+license-file = LICENSE
+license = MIT
+
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Science/Research
+    Intended Audience :: Developers
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+
+[options]
+packages = find:
+python_requires = >=3.8
+zip_safe = False
+
+install_requires =
```

Comment:
Do we want to put the dependencies that stuff like elastic beanstalk requires? I think it was this group:

```
uvloop = "*"
websockets = "*"
httptools = "*"
```

Reply:
No, these are only used for production deployment purposes